daisy_workflows/image_import/import_image.sh

```diff
@@ -21,9 +21,10 @@ SOURCE_DISK_FILE="$(curl -f -H Metadata-Flavor:Google ${URL}/attributes/source_d
 SOURCEURL=${SOURCE_DISK_FILE}
 SOURCEBUCKET="$(echo ${SOURCEURL} | awk -F/ '{print $3}')"
 SOURCEPATH="${SOURCEURL#"gs://"}"
-SOURCESIZE_BYTES="$(gsutil du ${SOURCEURL} | grep -o '^[0-9]\+')"
+SOURCE_FILE_TYPE="${SOURCEURL##*.}"
+SOURCE_SIZE_BYTES="$(gsutil du ${SOURCEURL} | grep -o '^[0-9]\+')"
 DISKNAME="$(curl -f -H Metadata-Flavor:Google ${URL}/attributes/disk_name)"
-STAGING_DISKNAME="$(curl -f -H Metadata-Flavor:Google ${URL}/attributes/staging_disk_name)"
+SCRATCH_DISKNAME="$(curl -f -H Metadata-Flavor:Google ${URL}/attributes/scratch_disk_name)"
```

Comment:
variable naming is not consistent: we have old variables without underscores and new that are a mix of underscores and no underscores. I have no preference for either but it should be consistent across the file

Reply:
Sounds good -- all of the variables that I added now use underscores